share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -27,7 +27,7 @@ default:
     - spack --version
     - cd share/spack/gitlab/cloud_pipelines/stacks/${SPACK_CI_STACK_NAME}
     - spack env activate --without-view .
-    - spack ci generate --check-index-only
+    - spack -d ci generate --check-index-only
```

Comment:
@tgamblin I feel like some of my changeds pushed with the squash may have been lost when you re-expanded the history..  This is the third thing I know I fixed already.  Unfortunately I did a hard reset to the remote version before checking carefully :facepalm: 

I don't suppose you have the squashed commit in your local history?